src/js/libgeo/GeoOps.js

```diff
@@ -1211,19 +1211,32 @@ geoOps.CircleBy3 = {};
 geoOps.CircleBy3.kind = "C";
 geoOps.CircleBy3.updatePosition = function(el) {
     var a = csgeo.csnames[(el.args[0])].homog;
+    el.startPoint = a;
     var b = csgeo.csnames[(el.args[1])].homog;
+    el.viaPoint = b;
     var c = List.ii;
     var d = List.jj;
     var p = csgeo.csnames[(el.args[2])].homog;
+    el.endPoint = p;
+
     var erg = geoOps._helper.ConicBy5(el, a, b, c, d, p);
     el.matrix = List.normalizeMax(erg);
     el.matrix = General.withUsage(el.matrix, "Circle");
 
 };
+geoOps.CircleBy3.initialize = function(el) {
+    el.startPoint = csgeo.csnames[(el.args[0])].homog;
+    el.viaPoint = csgeo.csnames[(el.args[1])].homog;
+    el.endPoint = csgeo.csnames[(el.args[2])].homog;
+};
```

Comment:
Why do we store this information with a circle that is not an arc? Seems like we shouldn't need it, and it might tempt some future contribution to use it inappropriately if we are not careful enough.